packages/wrangler/src/deploy/index.ts

```diff
@@ -217,6 +217,7 @@ export function deployOptions(yargs: CommonYargsArgv) {
 export async function deployHandler(
 	args: StrictYargsOptionsToInterface<typeof deployOptions>
 ) {
+	const beforeUpload = Date.now();
```

Comment:
This is probably not the right place to put this—the wrangler banner might fetch an updated Wrangler version from npm, which would influence this timing

Reply:
I've moved this line to right before the call to `deploy`, I figured that `requireAuth` and `standardPricingWarning` shouldn't be considered for this metric.